frontend/pages/policies/constants.ts

```diff
@@ -426,7 +426,7 @@ export const DEFAULT_POLICIES: IPolicyNew[] = [
   {
     key: 38,
     query:
-      "SELECT 1 WHERE NOT EXISTS (SELECT 1 FROM file WHERE filename like '%%Emergency Kit%%.pdf' AND (path LIKE '/Users/%%/Downloads/%%' OR path LIKE '/Users/%%/Desktop/%%'));",
+      "SELECT EXISTS(SELECT 1 FROM file WHERE filename like '%Emergency Kit%' AND (path LIKE '/Users/%%/Downloads/%%' OR path LIKE '/Users/%%/Desktop/%%')) as does_1p_ek_exist;",
```

Comment:
Same as above -- minor nit: We should add the `.pdf` so this would be: `WHERE filename LIKE '%Emergency Kit%.pdf'`

Reply:
Done ✅